lineapy/utils/utils.py

```diff
@@ -41,6 +41,8 @@ def get_literal_value_from_string(
         return val
     if literal_type == LiteralType.Ellipsis:
         return ...
+    if literal_type == LiteralType.Bytes:
+        return bytes(val, "utf-8")
```

Comment:
where does utf-8 get set when saving to db? is that something sqlalchemy adds or is that dependent on the backing database?

Reply:
This actually made me discover a bug. Our "serialization" logic is https://github.com/LineaLabs/lineapy/blob/c8c0e3011ebb7bfb8acc2aa19776257a47dad99f/lineapy/db/db.py#L270 
which is just `str(val)`. 

This code snippet actually shows the deserialization using utf-8 is wrong. WE should use eval, or ast.literal_eval instead. 

```
>>> x = str(b'abc')
>>> bytes(x, encoding='utf-8')
b"b'abc'"
>>> ast.literal_eval(x)
b'abc'
```

PR updated

Reply:
Added, also tested to make sure it fails with the wrong decoding.